.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,32 @@
+name: Go
+
+on:
+  pull_request:
+    branches: [development]
+    types: [opened, ready_for_review]
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15.2
```

Comment:
go1.15.5 ?  this is the minimum version in elrond scripts mainnet

Reply:
done.